Running configure with --without-securid causes failure because the 
securid_usersfile_find func isnt properly protected.  Fix sent upstream.

--- auth-securid.c
+++ auth-securid.c
@@ -65,6 +65,7 @@
  */
 #define INBUFFLEN 256
 
+#ifdef SECURID
 int
 securid_usersfile_find(const char *pw_name)
 {
@@ -104,6 +105,7 @@
 		pw_name, options.securid_usersfile);
 	return 0;
 }
+#endif
 int
 auth_securid(Authctxt *authctxt, const char *password)
 {
